src/ChronosTime.php

```diff
@@ -323,6 +337,37 @@ public function format(string $format): string
         return $this->toDateTimeImmutable()->format($format);
     }
 
+    /**
+     * Reset the format used to the default when converting to a string
+     *
+     * @return void
+     */
+    public static function resetToStringFormat(): void
```

Comment:
We probably never needed the reset helpers since you can pass the constant to set directly.

Reply:
I just made it consistent with the other 2 classes.